src/controllers/handlers/scene-admin-handlers/add-scene-admin-handler.ts

```diff
@@ -36,10 +36,18 @@ export async function addSceneAdminHandler(
   const place = await getPlace(isWorlds, realmName, parcel)
 
   const isOwner = isWorlds
-    ? await hasWorldPermission(authenticatedAddress, place.world_name!)
+    ? await hasWorldOwnerPermission(authenticatedAddress, place.world_name!)
     : await hasLandPermission(authenticatedAddress, place.positions)
 
-  const isAdmin = await sceneAdminManager.isAdmin(place.id, authenticatedAddress)
+  const hasStreamPermissionsOnWorld =
+    isWorlds && (await sceneFetcher.hasWorldStreamingPermission(authenticatedAddress, realmName))
+
+  let isAdmin = false
+  if (hasStreamPermissionsOnWorld) {
+    isAdmin = true
+  } else {
+    isAdmin = await sceneAdminManager.isAdmin(place.id, authenticatedAddress)
+  }
```

Comment:
What about something like:

```suggestion
  const isAdmin = hasStreamPermissionsOnWorld || (await sceneAdminManager.isAdmin(place.id, authenticatedAddress))
```